conf/messages.cy

```diff
@@ -159,37 +160,335 @@ helpWithYourSelfAssessmentTaxReturn.income_from_property = Fideo - Incwm o eiddo
+register_or_stopping.not_self_employed = Os nad ydych yn hunangyflogedig
+register_or_stopping.did_not_file_return = Os na wnaethoch gyflwyno Ffurflen Dreth y llynedd a bod yn rhaid i chi gyflwyno un eleni,
+register_or_stopping.sa1_link = defnyddiwch ffurflen SA1 i gofrestru ar gyfer Hunanasesiad (yn agor tab newydd).
+register_or_stopping.are_self_employed = Os ydych yn hunangyflogedig
+register_or_stopping.did_not_file_this_year = Os na wnaethoch gyflwyno Ffurflen Dreth y llynedd a bod yn rhaid i chi gyflwyno un eleni,
+register_or_stopping.register_link = cofrestrwch ar gyfer Hunanasesiad (yn agor tab newydd).
+register_or_stopping.partnership = Os ydych yn bartner neu’n bartneriaeth
+register_or_stopping_utr_reference = Bydd angen y Cyfeirnod Unigryw y Trethdalwr (UTR) arnoch ar gyfer y bartneriaeth a’ch UTR personol eich hun os oes gennych un eisoes.
+register_or_stopping.new_partnership = Os yw’n bartneriaeth newydd, bydd yn rhaid i’r partner enwebedig hefyd gofrestru’r bartneriaeth.
+register_or_stopping.more_about_registering = Dysgwch ragor am gofrestru fel partner neu bartneriaeth (yn agor tab newydd).
+register_or_stopping.sa_guidance = Arweiniad Hunanasesiad
+register_or_stopping.learn_sa = Dysgu am Hunanasesiad (yn agor tab newydd)
+register_or_stopping.registering_sa_video = Fideo – Cofrestru ar gyfer Hunanasesiad (yn agor tab newydd)
+register_or_stopping.registering_sa_transcript = Cofrestru ar gyfer Hunanasesiad: trawsgrifiad fideo
+register_or_stopping.stop_sa = Stopio Hunanasesiad
+register_or_stopping.you_must = Mae’n rhaid i chi
+register_or_stopping.tell_hmrc_link = roi gwybod i CThEM os nad oes angen i chi gyflwyno Ffurflenni Treth mwyach.
+register_or_stopping.because_you_are = Gallai hyn fod oherwydd:
+register_or_stopping.list_stopping_sa = eich bod yn rhoi’r gorau i hunangyflogaeth
+register_or_stopping.list_no_longer = nad ydych yn masnachu fel unig fasnachwr mwyach
+register_or_stopping.list_ending = eich bod yn dod â phartneriaeth fusnes i ben neu’n ei gadael
+register_or_stopping.send_final_return = Bydd angen i chi anfon Ffurflenni Treth terfynol a rhoi gwybod i gyflogeion eich bod yn cau’ch busnes.
+register_or_stopping.we_will_stop = Byddwn yn stopio’ch Hunanasesiad ac yn canslo’ch cyfraniadau Yswiriant Gwladol Dosbarth 2.
+register_or_stopping.stopping_class_2 = Gall stopio’ch cyfraniadau Yswiriant Gwladol Dosbarth 2 olygu eich bod yn colli rhai budd-daliadau.
+register_or_stopping.can_choose = Gallwch ddewis parhau i fod wedi’ch cofrestru ar gyfer y cyfraniadau hyn ar ôl stopio Hunanasesiad os ydych am wneud taliadau gwirfoddol. Bydd hyn yn eich galluogi i hawlio budd-daliadau fel Gofal Plant sy’n Rhydd o Dreth neu Lwfans Mamolaeth.
+register_or_stopping.get_help = Get help with this page
+
+
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#~~ Payment and Penalties
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+payment_and_penalties.title = Hunanasesiad: taliadau a chosbau
```

Comment:
I think that the accepted pattern is that titles and headings should be the same. That aside, on this Welsh message, the title is preceded by Hunanasesiad (Self Assessment) and the heading is not. But in the English version of these messages it is the other way around. Please go over the message files make them consistent across English/Welsh